src/powershell/public/Get-FinOpsToolkitVersions.ps1

```diff
@@ -0,0 +1,49 @@
+<#
+    .SYNOPSIS
+        Retrieves available version numbers of the FinOps Hub template.
+        
+    .PARAMETER Latest
+        Will only return the latest version number of FinOps Hub template.
+       
+    .PARAMETER Preview
+        Includes pre-releases.
+    
+    .EXAMPLE
+        Get-FinOpsToolkitVersions
+        
+        Returns all available released version numbers of FinOps Hub templates.
+        
+    .EXAMPLE
+        Get-FinOpsToolkitVersions -Latest
+        
+        Returns only the latest version number of the FinOps Hub templates.
+#>
+function Get-FinOpsToolkitVersions
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter()]
+        [switch]
+        $Latest,
+
+        [Parameter()]
+        [switch]
+        $Preview
+    )
+
+    $releaseUri = 'https://api.github.com/repos/microsoft/cloud-hubs/releases'
+    [array]$releases = Invoke-WebRequest -Uri $releaseUri -Verbose:$false | ConvertFrom-Json
```

Comment:
nit: You could merge the preview check into this line if you want. Not a big deal. Just a thought.
```suggestion
    [array]$releases = Invoke-WebRequest -Uri $releaseUri -Verbose:$false `
    | ConvertFrom-Json `
    | Where-Object {($Preview -eq $true) -or ($_.prerelease -eq $false)}
```